### PR TITLE
added read me

### PR DESCRIPTION
**Pull request by mikrowelt**
added some inprovments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+### /current_user.json 
+
+Returns current user object
+
+Exmaple: 
+
+request:
+```
+window.OpenEd.api.request('/current_user.json', null, callback)
```
Comment (LeonidMorozov) [inline README.md]: please, use `/users/me.json` endpoint instead. It was added for OAuth. We can change it to return user details depended to `scope`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    contribution_name: 'Pearson'
+}, callback)
+```
+response:
+```
+{"meta":
+    {"pagination":
+        {
+            "offset":10,
+            "total_entries":894208,
```
Comment (LeonidMorozov) [inline README.md]: number is wrong

File: README.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+Example:
+
+request:
+```
+window.OpenEd.api.request('/resources.json', {
+    offset: 10,
+    limit: 10,
+    descriptive: 'keyword search',
+    grade: 'k',
+    resource_type: 'game',
```
Comment (LeonidMorozov) [inline README.md]: Filters is not working. Take a look on results.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+Returns current user object
+
+Exmaple: 
+
+request:
+```
+window.OpenEd.api.request('/current_user.json', null, callback)
+```
+response:
+```
```
Comment (larsburgess) [inline README.md]: You can use `json` code fencing here, for example:

```json
{
    "current_user":{
        "first_name":"Andrew",
        "last_name":"Saenkov",
        "full_name":"Andrew Saenkov",
        "username":"andrew",
        "email":"andrew@example.com"
     }
}
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
+# opened-oauth.js
+## Getting started
+1. Create a callback route on your website (you can see an exmaple at callback.html)
+2. Get application key(redirect_uri must be an url that you created in step 1)
```
Comment (LeonidMorozov) [inline README.md]: Today we're creating new OAuth apps by demand. So, it should be clear for consumer how to get App creds.

**Comment by LeonidMorozov** [comment]
@mikrowelt , I'm done w/ PR review. Some comments there.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+Exmaple: 
+
+request:
+```
+window.OpenEd.api.request('/users/me.json', null, callback)
+```
+response:
+```json
+{   
+    "current_user":{
```
Comment (LeonidMorozov) [inline README.md]: there's no `current_user` container on `/users/me.json`. Please, check the response and update the readme file.

**Comment by LeonidMorozov** [comment]
We have to have `Validating the token` step before to use it.
It will be nice to have it in our lib by default in implicit flow.
But let's add this link for now.
http://technotes.iangreenleaf.com/posts/closing-a-nasty-security-hole-in-oauth.html

**Comment by LeonidMorozov** [comment]
@larsburgess , @vtrv101 , @adamblum , @lucascampbell .
Looks good for me, please, merge.

**Pull request merged**